docs/guides/admin/docs/configuration/basic.md

```diff
@@ -1,108 +1,114 @@
-secret.
+Step 3: Setting up Apache ActiveMQ Message Broker
+-------------------------------------------------
 
+Opencast requires Apache ActiveMQ to relay messages between micro-services.
+For configuration details, please follow the:
 
-Step 4: Setting up Apache ActiveMQ Message Broker
--------------------------------------------------
-<!-- _This is a very nonchalant and subdued description for how important this config probably is -->
-Since version 2.0, Opencast requires a running Apache ActiveMQ instance with a specific configuration.  The message
-broker is mostly run on the admin server of Opencast but can be run separately. It needs to be started before Opencast.
-For more details about the setup, have a look at the [Apache ActiveMQ configuration guide](message-broker.md).
+- [Apache ActiveMQ configuration guide](message-broker.md)
 
 
-Step 5: Database Configuration
+Step 4: Database Configuration
 ------------------------------
 
-Opencast uses an integrated H2 database by default. While you will find it perfectly functional, it has certain
-drawbacks:
+Opencast uses an integrated H2 database by default, which has certain drawbacks:
 
-* It is rather slow
 * It cannot be used for distributed set-ups
 * Upgrading Opencast with this database is not possible
 
-For testing, it is totally fine to keep the internal database, but you are highly encouraged to switch to a stand-alone
-database for productional use. For more information about database configuration, have a look at the [Database
-Configuration](database.md) section.
+For testing, it is fine to keep the internal database, but configure a stand-alone database for production.
+Details about the configuration can be found at:
 
-Step 6: Elasticsearch Configuration
+- [Database Configuration](database.md)
+
+
+Step 5: Elasticsearch Configuration
 -----------------------------------
 
-Opencast requires Elasticsearch.
-For more details about the setup, have a look at the [Elasticsearch configuration
-guide](../modules/searchindex/elasticsearch.md).
+Opencast requires Elasticsearch for quick full-text search and data caching.
+For more details about the setup, have a look at the:
+
+- [Elasticsearch configuration guide](../modules/searchindex/elasticsearch.md).
+
 
-Step 7: HTTPS Configuration
+Step 6: HTTPS Configuration
 ---------------------------
-<!-- _Every other instance of linking to another doc so far has been a "do now, but in another doc", 
-whereas here it is very much "not necessary for right now": what to do? -->
-An installation without HTTPS does not make much sense today.
-Thus, make sure to follow [a configuration guide for HTTPS](https/index.md).
 
-Step 8: Setting the Storage Directory (optional)
+Opencast requires configuration to:
+
+- Make Opencast available externally
+- Secure connections from/to Opencast
+
+For this, follow one of the
+
+- [configuration guides for HTTPS](https/index.md).
```

Comment:
Hm... Because this actually links to a single page "Serve Content Via HTTPS" which then links to the three options of configuration, it might be better to say `[HTTPS configuration guides listed here] (https/index.md).`

Reply:
I like to keep this as it is